rllib/evaluation/sampler.py

```diff
@@ -165,14 +188,19 @@ def __init__(self,
         self.obs_filters = obs_filters
         self.extra_batches = queue.Queue()
         self.perf_stats = _PerfStats()
+        self.sample_collector = None
+        if _use_trajectory_view_api:
+            self.sample_collector = _MultiAgentSampleCollector(
+                policies, callbacks)
```

Comment:
Nit: for clarity, prefer to assign a variable only once,
```
if foo:
   self.sample_collector = _Multi..()
else:
   self.sample_collector = None
```

here and elsewhere.

Reply:
done